packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -112,13 +133,25 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
     >
       {actionListHeaderChild}
       <StyledListBoxWrapper
+        sections={sectionData}
+        windowSize={5}
+        initialNumToRender={5}
+        bouncesZoom={false}
+        bounces={false}
+        keyExtractor={(item: any) => {
+          return item.value;
+        }}
```

Comment:
Not the best practice to use index as key especially for use-cases where item's indexes keep changing. Unlikely in our case but not impossible (would change during typeahead). Do we know what the value of `item.value` is going to be? I think it expects a number/string only
```suggestion
        keyExtractor={(item: any, index: number) => {
          return index;
        }}
```

Reply:
🤔 not using the index, I'm using item.value only which is the value which user passes to the ActionListItem

Reply:
Wtf 😆

Reply:
I think I meant, should we use index here because value's value could be unpredictable. Even though using index is not the best practice it might be safer than value. This "value" is what the consumer would pass as options, right?

Reply:
> This "value" is what the consumer would pass as options, right?

Yes that should also be fine, value should be stable across renders. 

Either way users should not change the value of any <ActionListItem /> in the runtime, doesn't make sense to do so either way. 

Imagine: 
```
<select>
  <option value={Math.random() > 0.5 ? "what" : "why?"}>hello</option>
  <option value="world">world</option>
</select>
```